frontend/src/v5/ui/components/shared/revisionDetails/components/revisionsListItem/revisionsListItem.component.tsx

```diff
@@ -62,6 +68,18 @@ export const RevisionsListItem = ({ revision, containerId, active = false }: IRe
 				{desc}
 			</RevisionsListItemText>
 			<RevisionsListItemButton onClick={toggleVoidStatus} status={voidStatus} />
+			<Tooltip
+				title={(
+					<FormattedMessage
+						id="revisionDetails.list.item.download.tooltip"
+						defaultMessage="Download revision"
+					/>
+				)}
+			>
+				<DownloadButton onClick={downloadRevision}>
+					<DownloadIcon />
+				</DownloadButton>
```

Comment:
This would suffice for downloading the revision
```suggestion
       <a href={downloadRevisionUrl(...)} >
				<DownloadButton>
					<DownloadIcon />
				</DownloadButton>
     </a>
```